github-config.yaml

```diff
@@ -54,6 +54,7 @@ orgs:
       - TreeinRandomForest
       - vpavlin
       - suppathak
+      - laxmisravyarudraraju

```

Comment:
Can you please move your entry up a bit, so the list of members is kept in alphabetical order? (just for easier housekeeping) Thank you